Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'config'
 gem 'countries'
 gem 'cssbundling-rails'
 gem 'database_cleaner'
+gem 'devise', '~> 4.9'
```

Comment:
Do we need to specify a version range or can we just leave it empty as 4.9.3 is the newest one anyway?